lib/modules/platform/bitbucket/index.spec.ts

```diff
@@ -1668,7 +1668,12 @@ describe('modules/platform/bitbucket/index', () => {
         .get('/2.0/repositories/some/repo/pullrequests/5')
         .reply(200, { reviewers: [reviewer] })
         .put('/2.0/repositories/some/repo/pullrequests/5')
-        .reply(200);
+        .reply(200, { id: 5 })
+        .get(`/2.0/repositories/some/repo/pullrequests`)
+        .query(true)
```

Comment:
what is this for? to ignore query parameters?

Reply:
No.

When cache is initialized for first time, it's internal method `init()` is called which populates the cache with list of PRs. 

In real runs, `getPrList()` initializes/pupulates the cache at the beginning. But in the tests, `updatePr` is called directly hence we need this mock.

Reply:
i mean the `query(true)`

Reply:
Yes, adding `query(true)` matches the request regardless of what query params are passed.